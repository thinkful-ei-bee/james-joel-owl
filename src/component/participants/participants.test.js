import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import {participants} from '../../store/store';

import Participants from '../participants/participants';

describe('<Participants />', () => {
    it('renders without crashing', () => {
        // Create a DOM element to render the component into
        const div = document.createElement('div');

        // Render the component
        // If something is wrong it will fail here
        ReactDOM.render(<Participants people={participants} />, div);

        // Clean up
        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('<Participants />', () => {
    it('renders without crashing', () => {
        // Render the component, as JSON
        const tree = renderer.create(<Participants people={participants} />).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot(); 
    });
});