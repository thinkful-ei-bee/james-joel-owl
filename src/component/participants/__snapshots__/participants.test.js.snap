// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Participants /> renders without crashing 1`] = `
<div
  className="Participants"
>
  <div
    className="Participant-Card"
  >
    <img
      alt="user avatar"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
    <div
      className="Participant-Card-Name"
    >
      Koren Templeton
    </div>
    <div
      className="Participant-Card-Details"
    >
      <div
        className="Participant-Card-Stage"
      >
        on stage
        <a
          href="#"
        >
          Boot
        </a>
      </div>
    </div>
  </div>
  <div
    className="Participant-Card"
  >
    <img
      alt="user avatar"
      src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
    />
    <div
      className="Participant-Card-Name"
    >
      Caty Flucker
    </div>
    <div
      className="Participant-Card-Details"
    >
      <div
        className="Participant-Card-Stage"
      >
        
        <a
          href="#"
        >
          Boot
        </a>
      </div>
    </div>
  </div>
  <div
    className="Participant-Card"
  >
    <img
      alt="user avatar"
      src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
    />
    <div
      className="Participant-Card-Name"
    >
      Frank Runciman
    </div>
    <div
      className="Participant-Card-Details"
    >
      <div
        className="Participant-Card-Stage"
      >
        
        <a
          href="#"
        >
          Boot
        </a>
      </div>
    </div>
  </div>
  <div
    className="Participant-Card"
  >
    <img
      alt="user avatar"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
    <div
      className="Participant-Card-Name"
    >
      Ashla Attwool
    </div>
    <div
      className="Participant-Card-Details"
    >
      <div
        className="Participant-Card-Stage"
      >
        on stage
        <a
          href="#"
        >
          Boot
        </a>
      </div>
    </div>
  </div>
</div>
`;
